odd_collector_sdk/grammar_parser/build_dataset_field.py

```diff
@@ -0,0 +1,258 @@
+from pathlib import Path
+from lark import Lark, Tree, Token
+from odd_models import DataSetField, DataSetFieldType, Type
+from oddrn_generator import Generator
+from ..logger import logger
+from .field_types import StructType, ArrayType, MapType, BasicType, UnionType, Field
+from ..utils.metadata import DefinitionType, extract_metadata
+from .exceptions import *
```

Comment:
Please specify imports or use alias for namespace, do not import everything, it can cause shadowing